Makefile

```diff
@@ -0,0 +1,52 @@
+MAJOR = $(shell grep "^\#define[[:space:]]*MAJOR" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+MINOR = $(shell grep "^\#define[[:space:]]*MINOR" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+PATCH = $(shell grep "^\#define[[:space:]]*PATCHLVL" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+BUILD = $(shell grep "^\#define[[:space:]]*BUILD" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+VERSION = v$(MAJOR).$(MINOR).$(PATCH).$(BUILD)
+PREFIX = cba
+BIN = @CBA_A3
+ZIP = CBA_A3
+ARMA = include
```

Comment:
It is a folder where the required Arma 3 unpacked files that are `#include`d in the code can be placed. This particular folder is `.gitignore`d with this PR.

I mainly added that because I don't have Arma 3 unpacked on my laptop which is running Linux, so I manually copied the required files in there for testing.

It can be changed to anything really, and you can pass it yourself to make with `make ARMA=P:/` for example. Armake also doesn't require P-drive, that's why I think it might not be the best to set it as default. Best would probably be to include the necessary files directly in the repository but I am not sure how BI feels about that.